modules/howtos/examples/subdoc.js

```diff
@@ -225,6 +226,29 @@ async function go() {
   ])
   // end::mutate-upsert-parents[]
 
+  console.log('mutate-store-semantics')
+  // tag::mutate-store-semantics[]
+  try {
+    // Mutate fields in a document that may or may not exist.
+    await collection.mutateIn(
+      'alice-123',
+      [
+        couchbase.MutateInSpec.insert('name', 'Alice'),
+        couchbase.MutateInSpec.upsert('email', 'alice@test.com'),
+      ],
+      {
+        storeSemantics: StoreSemantics.Upsert,
+      }
+    )
+  } catch (e) {
+    if (e instanceof couchbase.PathExistsError) {
```

Comment:
this is only for the insert on line 236, not the upsert on line 237?

Reply:
Yep `upsert` wouldn't throw an error as it would just create the path instead.